elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/OidcActivationProcessor.java

```diff
@@ -48,6 +56,23 @@ public void deploy(DeploymentPhaseContext phaseContext) throws DeploymentUnitPro
         OidcConfigService configService = OidcConfigService.getInstance();
         if (configService.isSecureDeployment(deploymentUnit) && configService.isDeploymentConfigured(deploymentUnit)) {
             addOidcAuthDataAndConfig(phaseContext, configService, webMetaData);
+        } else if (deploymentUnit.getAttachment(Attachments.DEPLOYMENT_ROOT) != null) {
+            try {
+                InputStream is = deploymentUnit.getAttachment(Attachments.DEPLOYMENT_ROOT).getRoot().getChild("WEB-INF/oidc.json").openStream();
+                String oidcConfigString = readFromInputStream(is);
+                Map<String, SimpleAttributeDefinition> unsupportedAttributesMap = new HashMap<>();
+                unsupportedAttributesMap.put(ElytronOidcDescriptionConstants.SCOPE, SecureDeploymentDefinition.SCOPE);
```

Comment:
The attribute name is also available from the attribute definition, e.g., `SecureDeploymentDefinition.SCOPE.getName()` could be used.

Reply:
Just wondering, maybe we could create a `List<SimpleAttributeDefinition> nonDefaultAttributes` in `SecureDeploymentDefinition.java`. Then we could iterate over each of these non default attributes here.

Reply:
I can make that change. 
Would it be better to create it inside ElytronOidcDescriptionConstants? Because as we would see in the request RFE, the new attributes are added to ProviderAttributeDefinitions. And if in the future additional attributes get added to other children resources, then they can all access `List<SimpleAttributeDefinition> NON_DEFAULT_ATTRIBUTES` from ElytronOidcDescriptionConstants.

Reply:
fixed!

Reply:
For shared attributes, we have a variable that is defined in `ProviderAttributeDefinition`  that also gets used by `SecureDeploymentDefintion`. So we'd likely need to introduce a variable for shared non default attributes when we get to that case.

Reply:
Sounds good! 
I will keep that in mind for WFLY-17143